### PR TITLE
MML adjust

### PR DESCRIPTION
**Pull request by Apofenas**

### CODE REVIEW

**Comment by Apofenas** [comment]
This one's fine?

**Comment by CookieNoob** [comment]
see https://github.com/FAForever/nomads/pull/337#issuecomment-433918298
and yes, I will merge this. For the future keep in mind to put related changes into a single commit, it makes it easier to keep track of them afterwards.

File: projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_proj.bp [hunk 1]
```diff
@@ -59,15 +59,15 @@ ProjectileBlueprint {
         HelpText = 0,
     },
     Physics = {
-        Acceleration = 0,
+	Acceleration = 3,
```
Comment (CookieNoob) [inline projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_proj.bp]: you changed the style from spaces to tabs here. Please keep it consistent with the rest of the code. (tabs -> spaces)

File: projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_proj.bp [hunk 2]
```diff
@@ -59,15 +59,15 @@ ProjectileBlueprint {
...
         DestroyOnWater = false,
         DestroyOnWaterAfterExitingWater = true,
-        InitialSpeed = 35,
-        MaxSpeed = 50,
-        MaxZigZag = 25,
+	InitialSpeed = 10,
```
Comment (CookieNoob) [inline projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_proj.bp]: here as well

**Pull request closed without merging**